### PR TITLE
adding solutions for warmups

### PR DESCRIPTION
**Pull request by RachellCalhoun**
I made the translations look better and added solutions to warmup exercises

### CODE REVIEW

File: lesson_2/solutions/common_letter_solution.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+		common[x] = common.get(x,0) + 1
+
+	maxkey = None
+	maxval = None
+	for key,val in common.items():
+	    if val > maxval:
+	        maxkey = key
+	        maxval = val
+
+	return maxkey, maxval
```
Comment (jinpark) [inline lesson_2/solutions/common_letter_solution.py]: this should be `[maxkey, maxval]` i think

File: lesson_2/solutions/reverse_solution.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
+# Write a function that takes in a string and returns a new
+# string with the letters in reverse order. Assume that the input will always be strings.
+#
+# Difficulty: easy.
+#Below are three examples of different solutions.
```
Comment (jinpark) [inline lesson_2/solutions/reverse_solution.py]: :+1: :+1: :+1:

File: lesson_2/solutions/reverse_solution.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+print('reverse("abc") == "cba": ' + str(reverse("abc") == "cba"))
+
+print('reverse("a") == "a": ' + str(reverse("a") == "a"))
+
+print('reverse("") == "": ' + str(reverse("") == ""))
+
+# These are tests to check that your code is working. After writing
+# your solution, they should all print true. Do not change these
+
+print('reverse("abc") == "cba": ' + str(reverse("abc") == "cba"))
```
Comment (jinpark) [inline lesson_2/solutions/reverse_solution.py]: you probably meant to change this to `reverse2` or `reverse3` right?

Comment (RachellCalhoun) [inline lesson_2/solutions/reverse_solution.py]: Yeah i'll change it.

**Comment by RachellCalhoun** [comment]
jin I changed some answers. also None was a problem, i think because of python3 so I changed maxval and maxkey to 0

File: lesson_2/solutions/gross_pay_overtime_solution.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+# 40시간 이상 일할 경우, 시급을 1.5배로 종업원에게 지급하는 급여 계산 프로그램을 input을 사용하여 작성하세요.
+#45시간의 근무시간과 10.50의 시급을 테스트를 위하여 사용하세요. (총급여는 498.75가 되어야 합니다.)
+#string을 읽어내는데는 input을, string을 숫자로 변환시키는데는 float()를 사용해야 합니다.
+
+#에러 확인에 신경쓰지 않아도 됩니다. 사용자가 입력하는 값들은 정확하다고 간주하세요.
+#도전해보기: 유효한 숫자임을 확인하는 코드를 작성 해 보세요.
+
+#not checking errors:
+
```
Comment (jinpark) [inline lesson_2/solutions/gross_pay_overtime_solution.py:17]: Do you want to uncomment this one and comment the one down below? I think thats how you have it for the rest of the solutions

**Comment by jinpark** [comment]
LGTM overall, one small thing thats not a big deal. Feel free to merge when you're ready

**Pull request merged**